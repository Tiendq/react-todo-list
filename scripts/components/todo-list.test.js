import React from "react";
import renderer from "react-test-renderer";
import TodoList from "./todo-list";

test("List has 2 items", () => {
  const items = [{
    id: 1,
    text: "Item 1",
    done: false
  }, {
    id: 2,
    text: "Item 2",
    done: true
  }];

  const component = renderer.create(
    <TodoList items={items} />
  );

  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
