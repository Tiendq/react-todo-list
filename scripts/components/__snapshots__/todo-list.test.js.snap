exports[`test List has 2 items 1`] = `
<ul
  className="todolist">
  <li
    className="form-check todoitem undone">
    <label
      className="form-check-label">
      <input
        className="form-check-input"
        onChange={[Function]}
        type="checkbox" />
       
      Item 1
    </label>
    <button
      className="btn btn-danger btn-sm"
      onClick={[Function]}
      type="button">
      x
    </button>
  </li>
  <li
    className="form-check todoitem done">
    <label
      className="form-check-label">
      <input
        className="form-check-input"
        onChange={[Function]}
        type="checkbox" />
       
      Item 2
    </label>
    <button
      className="btn btn-danger btn-sm"
      onClick={[Function]}
      type="button">
      x
    </button>
  </li>
</ul>
`;
